winter_django/view.py

```diff
@@ -42,20 +43,61 @@ def authenticate_header(self, request):
 
 
 def create_django_urls(api_class_with_routes: Type) -> List:
+    warnings.warn('Deprecated. Use create_django_urls_from_routes instead.', DeprecationWarning)
     component = Component.get_by_cls(api_class_with_routes)
     django_urls = []
 
     for url_path, routes in _group_routes_by_url_path(component.methods):
-        django_view = create_drf_view(api_class_with_routes, routes).as_view()
+        django_view = _create_drf_view(api_class_with_routes, routes).as_view()
+        winter_url_path = f'^{url_path}$'
+        methods = [route.method for route in routes]
+        django_url_path = rewrite_uritemplate_with_regexps(winter_url_path, methods)
+        for route in routes:
+            django_urls.append(url(django_url_path, django_view, name=route.method.full_name))
```

Comment:
nit: i would extract it as separate row `url(django_url_path, django_view, name=route.method.full_name)`